public/content/documentation/native/controls/slider.md

```diff
@@ -0,0 +1,275 @@
+## General Notes
+
+How to test a slider
+
+## Videos
+
+### iOS Voiceover
+<video controls>
+  <source src="media/video/native/slider/slider-iOSVoiceover.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+
+### Android Talkback
+<video controls>
+  <source src="media/video/native/slider/slider-AndroidTalkback.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+## Condensed
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a slider
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab, ctr+tab or arrow keys: Focus moves visibly to the input
+   - Right and left arrow-keys: Increase / decrease value one step
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves to the input
+   - Ios and android-swipe-up/down: Increase/decrease slider value one step
+   - Android-volume or swipe up/down: Increase/decrease slider value one step
+
+
+3. Listen to screenreader output on all devices
+
+   - Name: Name describes the purpose of the control and matches the visible label
+   - Role: Identifies itself as "adjustable" in iOS and "Slider" in Android
+   - Group: Group label with control, when possible to give the slider a programmatic name
+   - State: Expresses its current value, if applicable
+
+4. Test device settings
+
+   - Text resize: Text can resize up to 200% without losing information
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/slider](https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/slider)
+
+## Gherkin
+
+### #a11y - Native Accessibility Acceptance Criteria
```

Comment:
The spacing of the first two gherkin is different than the spacing of the last two gherkin - lets update them to be consistent.